src/libra_py/workflows/pyxaid2/namd.py

```diff
@@ -419,10 +477,12 @@ def run_namd(params):
         H_vib[i].real().show_matrix("res/_hvib_chi_"+str(i)+"_re" % ())
         H_vib[i].imag().show_matrix("res/_hvib_chi_"+str(i)+"_im" % ())
 
-    #========== Compute decoherence times  ===============
-    tau, decoh_rates = comn.decoherence_times(H_vib, 1)
 
+    #========== Scale H_vibs ===============
+    scale_H_vib(H_vib, params["Chi_dE"], params["NAC_dE"], params["sc_nac_method"])
```

Comment:
1. scaling should be optional
2. NAC_dE - the name is not intuitive, not clear to be at all what it might mean
3. Chi_dE - the name choice is also not the best (should not refer to "Chi" or any other symbols we chose to denote the states locally, instead the meaning of the name should be transparent to a new user)